lib/server.js

```diff
@@ -100,7 +100,7 @@ class S3Server {
 
 export default function main() {
     let clusters = _config.clusters || 1;
-    if (process.env.S3BACKEND) {
+    if (process.env.S3BACKEND === 'mem') {
```

Comment:
Note: As much as possible, I'd like to only bring the real fix required, rather than the full commit, if avoidable.